src/app/Shared/Services/Api.service.tsx

```diff
@@ -807,7 +819,9 @@ export class ApiService {
               }
             }
           }
-        }`)
+        }`,
+          { connectUrl: target.connectUrl, recordingName, labels: this.stringifyRecordingLabels(labels) }
```

Comment:
To keep consistent, would it be `recordingName: recordingName` ?

Reply:
It could be, this is just a JS/TS shorthand. It's probably safer to explicitly set the key in the object so that the parameter name can be refactored without changing the structure of the object, but we already do this in many other places too...